### ISSUE TITLE
Update request: emanote 1.0.2.0 → 1.2.0

### ISSUE THREAD

**Issue by carschandler** [NixOS/nixpkgs#254999]
- Package name: emanote
- Latest released version: 1.2.0
- Current version on the unstable channel: 1.0.2.0
- Current version on the stable/release channel: 1.0.2.0

- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

**Notify maintainers**

 @maralorn 

-----

Note for maintainers: Please tag this issue in your PR.

**Comment by carschandler** [comment]
Additionally, the current version is throwing an error due to
```
error: Package ‘heist-extra-0.2.0.0’ in /nix/store/1rp9mgpy83k4w11gi58hqfcqhk3nnahp-source/pkgs/development/haskell-modules/hackage-packages.nix:138369 is marked as broken, refusing to evaluate
```

**Comment by maralorn** [comment]
emanote will probably be fixed with the next haskellPackages bump in a few days.

**Comment by carschandler** [comment]
Still having this issue after updating flake inputs today:
```
Package ‘heist-1.1.1.1’ in /nix/store/09yvj6yyxspzfivv91bcxwrjxawpk1g2-source/pkgs/development/haskell-modules/hackage-packages.nix:142014 is marked as broken, refusing to evaluate
```

**Comment by srid** [comment]
`heist` is failing due to a test failure. I think that's fixed in 1.1.1.2:

https://github.com/snapframework/heist/commit/4606a18bdc955c4dd63cd46ee09a8f1b8d6c3052
